daemon/daemon.go

```diff
@@ -736,23 +766,29 @@ func NewDaemon(config *Config, registryService *registry.Service) (daemon *Daemo
 		return nil, err
 	}
 
+	distributionMetadataStore := dmetadata.NewFSMetadataStore(filepath.Join(imageRoot, "distribution"))
+
 	eventsService := events.New()
-	logrus.Debug("Creating repository list")
-	tagCfg := &graph.TagStoreConfig{
-		Graph:    g,
-		Key:      trustKey,
-		Registry: registryService,
-		Events:   eventsService,
-	}
-	repositories, err := graph.NewTagStore(filepath.Join(config.Root, "repositories-"+d.driver.String()), tagCfg)
+
+	tagStore, err := tag.NewTagStore(filepath.Join(imageRoot, "repositories.json"))
```

Comment:
Do we want to keep calling this `repositories.json`. ... do we want to call the manager tagStore if it actually doesn't have much to do with tags any more.